homework/nwd-nnw/nwdNww.hpp

```diff
@@ -1,11 +1,37 @@
 #pragma once
+#include <cmath>
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    
+    auto result = 0;
+    auto range = 0;
+
+    if (lhs == 0){
+        range = std::abs(rhs);
+    }else if(rhs == 0){
+        range = std::abs(lhs);
+    }else if(lhs > rhs){
+        range = std::abs(rhs);
+    }else{
+        range = std::abs(lhs);
+    }
+
+    for (auto i=1; i <= range; ++i){
+        if(lhs%i == 0 and rhs%i == 0){
+            result = i;
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..